internal/pipe/changelog/changelog.go

```diff
@@ -306,17 +305,16 @@ func sortEntries(ctx *context.Context, entries []string) []string {
 	if direction == "" {
 		return entries
 	}
-	result := make([]string, len(entries))
-	copy(result, entries)
```

Comment:
The slice copy here is unnecessary because `sortEntries` is the last function to be called in `buildChangelog`. So it is okay to sort the `entries` slice in-place.

https://github.com/goreleaser/goreleaser/blob/d2469666b8fe4c14bbce4c7a58f557350fd19d64/internal/pipe/changelog/changelog.go#L274-L279